### ISSUE TITLE
False positive triggering of FUNCTION_BOOLEAN_PREFIX

### ISSUE THREAD

**Issue by petertrr** [cqfn/diKTat#86]
```
[INFO]    [ktlint] C:\Users\pwx5333130\Documents\diKTat\diktat-rules\src\main\kotlin\org\cqfn\diktat\ruleset\rules\kdoc\KdocFormatting.kt:298:25: [17] functions that return the value of Boolean type should have <is> or <has> prefix: hasEmptyLineAfter
```
```
[INFO]    [ktlint] C:\Users\pwx5333130\Documents\diKTat\diktat-rules\src\main\kotlin\org\cqfn\diktat\ruleset\utils\FunctonASTNodeUtils.kt:21:13: [17] functions that return the value of Boolean type should have <is> or <has> prefix: hasParametersKDoc
```

```
[INFO]    [ktlint] C:\Users\pwx5333130\Documents\diKTat\diktat-rules\src\main\kotlin\org\cqfn\diktat\ruleset\utils\KDocUtils.kt:20:23: [17] functions that return the value of Boolean type should have <is> or <has> prefix: hasKnownKDocTag
```

```
[INFO]    [ktlint] C:\Users\pwx5333130\Documents\diKTat\diktat-rules\src\main\kotlin\org\cqfn\diktat\ruleset\utils\StringUtils.kt:23:12: [17] functions that return the value of Boolean type should have <is> or <has> prefix: isASCIILettersAndDigits
```

Probably because this is an extension function.
